api/types/container/container.go

```diff
@@ -138,6 +138,7 @@ type Summary struct {
 		NetworkMode string            `json:",omitempty"`
 		Annotations map[string]string `json:",omitempty"`
 	}
+	Health          HealthSummary
```

Comment:
Wondering if we want to make it `*HealthSummary` and make it `nil/null` instead of the `Status=none`.

Thoughts?

cc @moby/moby-committers

Reply:
Yes, we probably want it to be omitted for older API versions

Reply:
After discussing further with the other maintainers, we have reached the consensus that the HealthSummary should be omitted entirely from responses to requests with an older API version.
```suggestion
	Health          *HealthSummary `json:",omitempty"`
```